crates/turbopack/src/module_options/mod.rs

```diff
@@ -76,12 +76,25 @@ impl ModuleOptionsVc {
         };
 
         let css_transforms = CssInputTransformsVc::cell(vec![CssInputTransform::Nested]);
+        let mdx_transforms = EcmascriptInputTransformsVc::cell(
+            vec![EcmascriptInputTransform::TypeScript]
```

Comment:
Should this be gated to `enable_typescript_transform`?

Reply:
This is part I'm unusure. MDX can accept typescript, but it is orthogonal to plain jsx | tsx config afaik. I choose to enable ts by default for the safety.